pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -515,7 +515,7 @@ func updateConfigTimer(configInterval uint32, tickerHandle interface{}) {
 
 // Called when globalConfig changes
```

Comment:
This comment is no longer correct. I think you can drop this line unless you want to describe the current usage.